images/tls-certificates.md

```diff
@@ -38,3 +41,53 @@ within the workspace will use the certificates when making requests.
 The specific process to add system-level certificates depends on the Linux
 distribution that you're using, but it is typically done by adding your
 certificates to your system's trusted CA repository.
+
+## TLS Certificate Requirements
+
+Since the publication of RFC 2818 in 2000, the `commonName` field has been
+[considered deprecated](https://groups.google.com/a/chromium.org/g/security-dev/c/IGT2fLJrAeo/m/csf_1Rh1AwAJ).
+
+The Go programming language, which Coder uses, recently began enforcing this and
+ignoring the `commonName` field (source) in favour of the Subject Alternative
+Name (SAN) field.
+
+This essentially means that SSL certificates are required to use
+`Subject Alternative Name` instead of `commonName`. If you attempt to use such a
+certificate with Coder, you may see the following error:
```

Comment:
Suggestion: Unclear such-reference (IMO):

```suggestion
`Subject Alternative Name` instead of `commonName`. If you attempt to use a
certificate having `commonName` with Coder, you may see the following error:
```